### PR TITLE
Fix typo in remove-controls-3.1.md

### PR DESCRIPTION
**Pull request by lutzroeder**
@gewarren @RussKie

### CODE REVIEW

File: includes/core-changes/windowsforms/3.1/remove-controls-3.1.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ Each removed control has a recommended replacement control. Refer to the followi
 | ContextMenu | ContextMenuStrip | |
 | Menu | ToolStripDropDown, ToolstripDropDownMenu | MenuItemCollection |
```
Comment (RussKie) [inline includes/core-changes/windowsforms/3.1/remove-controls-3.1.md]:
```suggestion
| Menu | ToolStripDropDown, ToolStripDropDownMenu | MenuItemCollection |
```

**Review by RussKie — APPROVED** [review]

File: includes/core-changes/windowsforms/3.1/remove-controls-3.1.md (hunk 1 above)
Comment (lutzroeder) [reply includes/core-changes/windowsforms/3.1/remove-controls-3.1.md]: @tdykstra do you have permission to commit this?

**Review by tdykstra — APPROVED** [review]
LGTM, thanks for the correction, @lutzroeder

File: includes/core-changes/windowsforms/3.1/remove-controls-3.1.md (hunk 1 above)
Comment (tdykstra) [reply includes/core-changes/windowsforms/3.1/remove-controls-3.1.md]: https://github.com/dotnet/docs/pull/16764

**Pull request merged**